There were tests whose specified size is too big. Use the --test_verbose_timeout_warnings command line option to see which ones these are.

https://github.com/enola-dev/enola/pull/1778 failed like this:

```
(...)

[2,962 / 2,966] 76 / 79 tests; Testing //java/dev/enola/web:tests; 10s disk-cache, processwrapper-sandbox ... (4 actions running)
[2,965 / 2,966] 78 / 79 tests; Testing //java/dev/enola/cli:tests; 3s disk-cache, processwrapper-sandbox
[2,965 / 2,966] 78 / 79 tests; Testing //java/dev/enola/cli:tests; 13s disk-cache, processwrapper-sandbox
FAIL: //java/dev/enola/cli:tests (Exit 1) (see /home/runner/.bazel/execroot/_main/bazel-out/k8-fastbuild/testlogs/java/dev/enola/cli/tests/test.log)
[2,965 / 2,966] 79 / 79 tests, 1 failed; Testing //java/dev/enola/cli:tests; 40s disk-cache, processwrapper-sandbox
INFO: From Testing //java/dev/enola/cli:tests:
stdout (/home/runner/.bazel/execroot/_main/bazel-out/_tmp/actions/stdout-2966) 3186106 exceeds maximum size of --experimental_ui_max_stdouterr_bytes=1048576 bytes; skipping
INFO: Found 200 targets and 79 test targets...
INFO: Elapsed time: 115.104s, Critical Path: 48.97s
INFO: 2966 processes: 1337 disk cache hit, 1410 internal, 119 processwrapper-sandbox, 100 worker.
INFO: Build completed, 1 test FAILED, 2966 total actions
//java/dev/enola/common/exec:tests                              (cached) PASSED in 1.1s
//java/dev/enola/common/exec/vorburger:tests                    (cached) PASSED in 1.4s
//java/dev/enola/common/io/metadata:tests                       (cached) PASSED in 0.5s
//java/dev/enola/common/io/util:tests                           (cached) PASSED in 0.6s
//java/dev/enola/common/linereader:tests                        (cached) PASSED in 0.8s
//java/dev/enola/common/markdown:tests                          (cached) PASSED in 1.0s
//java/dev/enola/common/markdown/exec:tests                     (cached) PASSED in 1.7s
//java/dev/enola/common/name:tests                              (cached) PASSED in 0.9s
//java/dev/enola/common/text:tests                              (cached) PASSED in 0.6s
//java/dev/enola/connect/maven:tests                            (cached) PASSED in 10.9s
//java/dev/enola/ai/adk/core:tests                                       PASSED in 2.0s
//java/dev/enola/ai/adk/demo:tests                                       PASSED in 1.7s
//java/dev/enola/ai/adk/iri:tests                                        PASSED in 2.7s
//java/dev/enola/ai/adk/test:tests                                       PASSED in 1.8s
//java/dev/enola/ai/adk/tool:tests                                       PASSED in 2.0s
//java/dev/enola/ai/adk/web:tests                                        PASSED in 8.5s
//java/dev/enola/ai/dotagent:tests                                       PASSED in 2.5s
//java/dev/enola/ai/dotprompt:tests                                      PASSED in 2.1s
//java/dev/enola/ai/dotprompt/adk:tests                                  PASSED in 2.7s
//java/dev/enola/ai/iri:tests                                            PASSED in 0.7s
//java/dev/enola/ai/langchain4j:tests                                    PASSED in 1.3s
//java/dev/enola/ai/mcp:tests                                            PASSED in 1.8s
//java/dev/enola/cas:tests                                               PASSED in 4.7s
//java/dev/enola/chat:tests                                              PASSED in 1.7s
//java/dev/enola/chat/sshd:tests                                         PASSED in 2.0s
//java/dev/enola/cli:EnolaLoggingTest                                    PASSED in 2.8s
//java/dev/enola/common:tests                                            PASSED in 0.8s
//java/dev/enola/common/canonicalize:tests                               PASSED in 0.8s
//java/dev/enola/common/concurrent:tests                                 PASSED in 0.8s
//java/dev/enola/common/context:tests                                    PASSED in 1.1s
//java/dev/enola/common/context/testlib:tests                            PASSED in 0.7s
//java/dev/enola/common/convert:tests                                    PASSED in 1.2s
//java/dev/enola/common/diff/testlib:tests                               PASSED in 0.7s
//java/dev/enola/common/exec/pty:tests                                   PASSED in 2.0s
//java/dev/enola/common/html:tests                                       PASSED in 0.7s
//java/dev/enola/common/io:tests                                         PASSED in 9.2s
//java/dev/enola/common/io/iri:tests                                     PASSED in 1.2s
//java/dev/enola/common/io/object:tests                                  PASSED in 0.8s
//java/dev/enola/common/io/object/jackson:tests                          PASSED in 2.2s
//java/dev/enola/common/io/testlib:tests                                 PASSED in 1.0s
//java/dev/enola/common/locale:tests                                     PASSED in 0.7s
//java/dev/enola/common/protobuf:tests                                   PASSED in 1.6s
//java/dev/enola/common/protobuf/schema:tests                            PASSED in 0.7s
//java/dev/enola/common/secret:tests                                     PASSED in 0.7s
//java/dev/enola/common/secret/auto:tests                                PASSED in 1.4s
//java/dev/enola/common/secret/exec:tests                                PASSED in 0.7s
//java/dev/enola/common/secret/gnome:tests                               PASSED in 0.8s
//java/dev/enola/common/secret/yaml:tests                                PASSED in 0.8s
//java/dev/enola/common/string2long:tests                                PASSED in 0.9s
//java/dev/enola/common/template:tests                                   PASSED in 0.7s
//java/dev/enola/common/template/handlebars:tests                        PASSED in 1.7s
//java/dev/enola/common/time:tests                                       PASSED in 0.8s
//java/dev/enola/common/tree:tests                                       PASSED in 0.7s
//java/dev/enola/common/xml:tests                                        PASSED in 1.4s
//java/dev/enola/common/yamljson:tests                                   PASSED in 1.1s
//java/dev/enola/common/yamljson/testlib:tests                           PASSED in 0.9s
//java/dev/enola/core:tests                                              PASSED in 5.5s
//java/dev/enola/data/id:tests                                           PASSED in 1.0s
//java/dev/enola/data/iri:tests                                          PASSED in 0.8s
//java/dev/enola/data/iri/namespace:tests                                PASSED in 0.6s
//java/dev/enola/data/iri/namespace/repo:tests                           PASSED in 0.8s
//java/dev/enola/data/iri/template:tests                                 PASSED in 1.1s
//java/dev/enola/datatype:tests                                          PASSED in 0.7s
//java/dev/enola/format/tika:tests                                       PASSED in 2.4s
//java/dev/enola/format/tika/rdf:tests                                   PASSED in 2.1s
//java/dev/enola/format/xml:tests                                        PASSED in 1.1s
//java/dev/enola/identity:tests                                          PASSED in 1.1s
//java/dev/enola/infer/datalog:tests                                     PASSED in 0.7s
//java/dev/enola/infer/rdf:tests                                         PASSED in 1.0s
//java/dev/enola/model:tests                                             PASSED in 4.1s
//java/dev/enola/rdf/io:tests                                            PASSED in 3.5s
//java/dev/enola/rdf/proto:tests                                         PASSED in 1.1s
//java/dev/enola/thing:tests                                             PASSED in 2.0s
//java/dev/enola/thing/gen:tests                                         PASSED in 3.7s
//java/dev/enola/thing/testlib:tests                                     PASSED in 0.6s
//java/dev/enola/web:tests                                               PASSED in 10.8s
//java/dev/enola/web/netty:tests                                         PASSED in 6.2s
//java/dev/enola/zimpl:EnolaTest                                         PASSED in 1.0s
//java/dev/enola/cli:tests                                               FAILED in 40.8s
  /home/runner/.bazel/execroot/_main/bazel-out/k8-fastbuild/testlogs/java/dev/enola/cli/tests/test.log
Executed 69 out of 79 tests: 78 tests pass and 1 fails locally.
There were tests whose specified size is too big. Use the --test_verbose_timeout_warnings command line option to see which ones these are.
Error: Process completed with exit code 123.
```

I cannot reproduce this locally, and I'm having a hard time understanding the root cause on CI based on this output.

I'll see if I can add that `--experimental_ui_max_stdouterr_bytes=-1` and `--test_verbose_timeout_warnings` command line options, and print ` /home/runner/.bazel/execroot/_main/bazel-out/k8-fastbuild/testlogs/java/dev/enola/cli/tests/test.log`.

--- a/java/dev/enola/rdf/io/RdfResourceIntoProtoThingConverter.java
+++ b/java/dev/enola/rdf/io/RdfResourceIntoProtoThingConverter.java
@@ -51,7 +51,7 @@ public class RdfResourceIntoProtoThingConverter
         return Optional.of(rdfThingConverter.convertToList(optModel.get()));
     }
 
-    /** This return thingsList as Thing (if there is 1) or a {@link Things} pb. */
+    /** This returns thingsList as Thing (if there is 1) or a {@link Things} pb. */
     public Message.Builder asMessage(List<Thing.Builder> thingsList) {
         Message.Builder messageBuilder;
         if (thingsList.size() == 1) messageBuilder = thingsList.get(0);

--- a/test.bash
+++ b/test.bash
@@ -72,7 +72,9 @@ else # On CI
   tools/git/test.bash
 
   # Test all Bazel targets
-  "$BZL" query //... | xargs "$BZL" test
+  # https://github.com/enola-dev/enola/issues/1780
+  "$BZL" query //... | xargs "$BZL" test --experimental_ui_max_stdouterr_bytes=-1
+  cat ~/.bazel/execroot/_main/bazel-out/k8-fastbuild/testlogs/java/dev/enola/cli/tests/test.log || true
 fi
 
 ./test-cli.bash

